source/_integrations/uk_transport.markdown

```diff
@@ -31,6 +31,7 @@ sensor:
   - platform: uk_transport
     app_id: YOUR_APP_ID
     app_key: YOUR_APP_KEY
+    scan_interval: 86400
```

Comment:
Scan internal is optional (and thus not part of any initial example) and not related to this integration. As a matter of fact, we discourage the use of scan interval completely, as it more often leads to setting it higher than needed with resulting in high request volumes on third-party services.

It is for that same reason, scan intervals are complete phased out throughout Home Assistant (one of the reasons).

Lastly, this is a platform option, and documented in the platform options in the documentation.

Going to friendly decline and close the PR for the above reasoning.

../Frenck

Reply:
Not a problem. It is the exact opposite though, the higher the scan_interval, the fewer the API requests and this is extremely useful for users of the free tier.

Reply:
In that case, we should adjust the default to be more sane :)